be/src/runtime/fragment_mgr.cpp

```diff
@@ -591,12 +591,16 @@ void FragmentMgr::report_fragments_with_same_host(
 
             if (executor->is_done()) {
                 reported[i] = true;
```

Comment:
Why unregister the profile when the executor done?

Reply:
At the end of fragment instance execution( function '_open_internal_vectorized'), BE will send_report to FE through coordinator_callback function. 
ProfileReportWorker is not necessary to send report to FE at the end of fragment instance.